packages/components/src/lib/inputUtils.ts

```diff
@@ -0,0 +1,21 @@
+import { SqInput } from "@quri/squiggle-lang";
+
+// Some input defaults are not strings, but we need to pass strings to Squiggle.
+export function defaultAsString(row: SqInput): string {
```

Comment:
We need to convert these to strings anyway, to run through Squiggle (I guess we could have a separate pipeline where we convert booleans directly into SqBoolean, but that would be tricker), so it didn't seem too bad to convert it at this stage.